src/lib.rs

```diff
@@ -1,9 +1,16 @@
 #![allow(non_snake_case, dead_code)]
 
-use raxb::{value::ConstStr, XmlDeserialize, XmlSerialize};
+use std::default;
+
+use _raxb::quick_xml::{events::Event, name::ResolveResult};
+use raxb::ty::S;
+use raxb::{de::XmlDeserializeError, value::ConstStr, XmlDeserialize, XmlSerialize};
 use serde::{Deserialize, Serialize};
-use wasm_bindgen::prelude::*;
 use tsify::Tsify;
+use wasm_bindgen::prelude::*;
+
+// #[cfg(test)]
```

Comment:
No commented out code. If this should be an issue in the future, create issue instead and reference original issue in it.